scrimage-webp/src/main/java/com/sksamuel/scrimage/webp/WebpHandler.java

```diff
@@ -32,6 +32,21 @@ protected static void installBinary(Path output, String... sources) throws IOExc
       throw new IOException("Could not locate webp binary at " + Arrays.toString(sources));
    }
 
+   protected static String[] getBinaryPath(String binaryName) {
+      String os = "linux";
+
+      if(SystemUtils.IS_OS_WINDOWS) {
+         os = "window";
+      } else if(SystemUtils.IS_OS_MAC) {
+         os = "mac";
+      }
+
+      return new String[] {
+         "/webp_binaries/"      + os + "/" + binaryName,
```

Comment:
Do we need to include the OS in the target path, since you're not going to have more than one installed at a time? And this way it would rename backwards compatible ?

Reply:
i think i misunderstood. it definitly does have backward compatilbity issue.

loading webp binary as environment variable looks better.